### ISSUE TITLE
Conflict between prettier and stylelint while working with @font-face and multiple sources

### ISSUE THREAD

**Issue by silversonicaxel** [Jamie-Yang/vue3-boilerplate#8]
Considering an `App.vue` like this one

```
<template>
  <div id="app">
    <router-view />
  </div>
</template>

<style lang="scss">
@font-face {
  font-family: 'Share Tech';
  src: url('../src/assets/fonts/ShareTech-Regular.eot');
  src:
    url('../src/assets/fonts/ShareTech-Regular.eot?#iefix') format('embedded-opentype'),
    url('../src/assets/fonts/ShareTech-Regular.woff2') format('woff2'),
    url('../src/assets/fonts/ShareTech-Regular.woff') format('woff'),
    url('../src/assets/fonts/ShareTech-Regular.ttf') format('truetype'),
    url('../src/assets/fonts/ShareTech-Regular.svg#ShareTech-Regular') format('svg');
  font-weight: normal;
  font-style: normal;
  font-display: swap;
}
</style>
```
here prettier of complains
```
<e>      src/App.vue:15:7
<e>      13 |   font-family: 'Share Tech';
<e>      14 |   src: url('../src/assets/fonts/ShareTech-Regular.eot');
<e>    > 15 |   src:
<e>         |       ^
<e>      16 |     url('../src/assets/fonts/ShareTech-Regular.eot?#iefix') format('embedded-opentype'),
<e>      17 |     url('../src/assets/fonts/ShareTech-Regular.woff2') format('woff2'),
<e>      18 |     url('../src/assets/fonts/ShareTech-Regular.woff') format('woff'),
<e>
<e> ✘ 1 problem (1 error, 0 warnings)
<e>
<e>
<e> Errors:
<e>   1  https://google.com/#q=prettier%2Fprettier
<e>
```

Considering the same `App.vue` like this one
```
<template>
  <div id="app">
    <router-view />
  </div>
</template>

<style lang="scss">
@font-face {
  font-family: 'Share Tech';
  src: url('../src/assets/fonts/ShareTech-Regular.eot');
  src: url('../src/assets/fonts/ShareTech-Regular.eot?#iefix') format('embedded-opentype'),
    url('../src/assets/fonts/ShareTech-Regular.woff2') format('woff2'),
    url('../src/assets/fonts/ShareTech-Regular.woff') format('woff'),
    url('../src/assets/fonts/ShareTech-Regular.ttf') format('truetype'),
    url('../src/assets/fonts/ShareTech-Regular.svg#ShareTech-Regular') format('svg');
  font-weight: normal;
  font-style: normal;
  font-display: swap;
}
</style>
```
then stylelint complains
```
$ stylelint "./src/**/*.vue" "./src/**/*.scss"

src/App.vue
 15:6  ✖  Expected newline after ":" with a multi-line declaration   declaration-colon-newline-after
```

**Comment by Jamie-Yang** [comment]
Thank you for reporting this issues.
It has been fixed, view changes: [a6465ab](https://github.com/Jamie-Yang/vue3-boilerplate/commit/a6465ab280015dae0a069aa08e52cc55a45b8a2a)

**Comment by silversonicaxel** [comment]
awesome job!
